src/parser/monk/brewmaster/integrationTests/example.test.ts.snap

```diff
@@ -1381,7 +1381,7 @@ exports[`Brewmaster Monk integration test: example log CastEfficiency matches th
               }
             >
               22
-              /36
+              /32
```

Comment:
We will need a new test log for this. Looks like the old one does not have the new talent ID.